<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="sourcesTree">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="itemsExpandable">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="newDirectoryAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/folder_add.png</normaloff>:/resources/fatcow16x16/folder_add.png</iconset>
   </property>
   <property name="text">
    <string>New director&amp;y...</string>
   </property>
  </action>
  <action name="newFileAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/page_white_add.png</normaloff>:/resources/fatcow16x16/page_white_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New file...</string>
   </property>
  </action>
  <action name="renameAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/text_edit.png</normaloff>:/resources/fatcow16x16/text_edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename...</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/bin_closed.png</normaloff>:/resources/fatcow16x16/bin_closed.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="refreshAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/arrow_refresh.png</normaloff>:/resources/fatcow16x16/arrow_refresh.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;fresh</string>
   </property>
  </action>
  <action name="duplicateAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/calendar_copy.png</normaloff>:/resources/fatcow16x16/calendar_copy.png</iconset>
   </property>
   <property name="text">
    <string>D&amp;uplicate</string>
   </property>
  </action>
  <action name="editProjectSettingsAction">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/resources/fatcow16x16/book_edit.png</normaloff>:/resources/fatcow16x16/book_edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit project settings...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sourcesTree</tabstop>
 </tabstops>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
